styles.css

```diff
@@ -129,33 +118,33 @@ a{
 	opacity:0.7;
 }
 
-.logo-1,
-.logo-2 {
+.logo-CSS,
```

Comment:
class names not consistant with others - snakeCase instead of hyphenated

Reply:
logo- used as a selector lines 190 and 197 as "logo" would select other elemtents

Reply:
eg logoBox

Reply:
would suggest changing to logoFor
eg: logoForCss, logoForPHP
so selector would be on logoFor
this would solve the prob and keep naming convention